INSTALL.md

```diff
@@ -36,7 +36,7 @@ When running a node, the best bet is to go with the latest release.
     # sudo apt-get install software-properties-common
     # sudo add-apt-repository ppa:ubuntu-toolchain-r/test
     # sudo apt-get update
-    # sudo apt-get install git build-essential pkg-config libtool autoconf automake libtool bison flex libpq-dev clang++-5.0 gcc-5 g++-5 cpp-5
+    # sudo apt-get install git build-essential pkg-config libtool autoconf automake libtool bison flex pandoc libpq-dev clang++-5.0 gcc-5 g++-5 cpp-5
```

Comment:
This may be the right time to split things up a bit better:
* `pandoc` is really optional for building `stellar-core`, you should put its install on its own line (`pandoc` is a large download)
* we should also split `gcc` vs `clang` installs (as most people only build with one or the other), leaving all the common stuff on its own line